files/routes/static.py

```diff
@@ -52,6 +52,11 @@ def sidebar(v):
 def rules(v):
 	return render_template('rules.html', v=v)
 
+@app.get('/support')
+@auth_desired
```

Comment:
Non-blocking: What does `auth_desired()` do here? Looking at the code it looks like it checks if the logged-in user is ban evading and shadowbans them if so, and is a noop if nobody is logged in, but I'm not clear on why it does that.

Reply:
I believe it loads the user if they're logged in, which allows the header menus to populate properly for their account.

Reply:
I'm gonna be honest, I just copy-pasted it from /rules.